Dockerfile

```diff
@@ -17,10 +18,12 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
     python3-setuptools \
     software-properties-common \
     unzip \
+    tar \
     wget \
     zip \
     && rm -rf /var/lib/apt/lists/*
 
 # upgrade gcc-arm-none-eabi from the default 5.4.1 to 6.3.1 due to ARM runtime issues
-RUN wget -q https://developer.arm.com/-/media/Files/downloads/gnu-rm/6-2017q2/gcc-arm-none-eabi-6-2017-q2-update-linux.tar.bz2 -O - | \
-    tar xj --strip-components=1 -C /
```

Comment:
I get this when I try to build:

    /bin/sh: 1: set: Illegal option -o pipefail

Reply:
I should have some time tomorrow to try and track this down.

Reply:
Rebased, and Travis has now built the image with the fix 07d52ba8ae64004affd896eb3e9ca96d94f1da41.